.github/workflows/packaging.yaml

```diff
@@ -94,3 +97,87 @@ jobs:
         run: |
           ./packaging/msi/winget_manifest.sh > Tetrate.func-e.yaml
           yamllint -sd '{extends: default, rules: {line-length: disable}}' Tetrate.func-e.yaml
+
+      - name: "Upload artifacts"
+        if: runner.os == 'Linux'
+        uses: actions/upload-artifact@v2
+        with:
+          name: dist
+          path: dist
+
+  linux-x86:
+    name: "Test installing on (${{ matrix.image }}/${{ matrix.arch }})"
+    needs: dist
+    runs-on: "ubuntu-latest"
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - image: "ubuntu:latest"
+            arch: amd64
+            packaging: deb
+          - image: "ghcr.io/tetratelabs/func-e-internal:centos8"
+            arch: x86_64
+            packaging: rpm
+
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v2
+
+      - name: "Download artifacts"
+        uses: actions/download-artifact@v2
+        with:
+          name: dist
+          path: dist
+
+      - name: "Install .${{ matrix.packaging }} on ${{ matrix.image }} (${{ matrix.arch }})"
+        run: |
+          docker run -e ARCH=${{ matrix.arch }} \
+            -e PACKAGING=${{ matrix.packaging }} \
+            -v ${{ github.workspace }}:/src \
+            -w /src \
+            ${{ matrix.image }} \
+            packaging/nfpm/verify.sh
+
+  linux-non-x86:
```

Comment:
this is pretty nasty

We are currently using travis to test arm64, let's please not use docker explicitly and drift how things are tested feature by feature. Ack you probably haven't seen enough of what's going on, yet, but it is a lot easier to be consistent in approach vs have one change be different only for one thing, as we'd have to remember even more context when doing things.